pipelines/managed/e2e/e2e.yaml

```diff
@@ -45,6 +45,10 @@ spec:
     - name: verify_ec_task_bundle
       type: string
       description: The location of the bundle containing the verify-enterprise-contract task
+    - name: verify_ec_task_git_revision
```

Comment:
I assume you will create a second PR later that will actually start calling the ec task via git resolver, right? Is there any benefit in doing it in two stages? Can't we just merge (and promote to production) https://github.com/konflux-ci/release-service/pull/715 and then do all the changes in the pipelines in one PR? It's ok pass parameters that the pipeline doesn't need.

Reply:
> I assume you will create a second PR later that will actually start calling the ec task via git resolver, right? 

yes. the trusted artifacts based pipelines

> Is there any benefit in doing it in two stages?

Initially I believed it had to be done in 2 steps. I was basing that on using the `tkn` command:

```
% tkn pipeline start scoheb-debug -pnew-parameter=s
Error: param 'new-parameter' not present in spec
```

Reply:
Oh, interesting. Fine with me to do it in stages.